### DISCUSSION TITLE
[Feature Request] /cmi enchant -lock

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#4542] [Feature Requests]
**Description of feature/improvement request:**

Ability to prevent further enchanting of an item. 

---
### Detailed explanation of what it should do, when and in what situation: ###

We have a few event items, unique items, and vote items that would be nice if they could stay as they are. Or maybe a one time use. Preventing further repair, but at least preventing further enchants. If we give someone a shovel with e8 on it, and then lock it. If the player would then try to add mending to it that cmi would say: Sorry yo, that is a no go.

### Basic explanation as to why this feature is needed and/or useful to have: ###

Unique items remain more unique, and semi op rewards can't be turned into rediculously enchant on top of enchant etc. For example, we have a silk touch 5 pick for dropping only a few spawner types. We don't want people adding mending and unbreaking to this pickaxe to make it last longer. The wood pick should run out of durability without being able to get repaired or further enchanted. 

Thank you for considering.

**Comment by CloudeLecaw** [comment]
i also would like to see this feature.
We have a lot of special Even items rewards, where er dont want player to enchant them with extra enchants. 
so please add this lock feature <3
